modules/ipi-install-configuring-networking.adoc

```diff
@@ -6,109 +6,166 @@
-    nmcli connection add ifname baremetal type bridge <con_name> baremetal bridge.stp no <1>
-    nmcli con add type bridge-slave ifname \"$PUB_CONN\" master baremetal
-    pkill dhclient;dhclient baremetal
-"
-----
-<1> Replace `<con_name>` with the connection name.
-
-.. For a network using static IP addressing and no DHCP network, run the following command:
++
+.. For a network using static IP addressing and no DHCP network, create an NMState YAML file and specify the bare-metal bridge interface details in the file:
++
+.Example bare-metal bridge interface configuration that uses static IP addressing and no DHCP network
+[source,yaml]
+----
+# ...
+dns-resolver: <1>
+  config:
+    server:
+      - <dns_ip_address> <2>
+interfaces:
+  - name: baremetal
+    type: linux-bridge <3>
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: false
+      address:
+        - ip: <static_ip_address>
+          prefix-length: 24
+      gateway: <gateway_ip> <4>
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port:
+        - name: <physical_interface_name>
+  - name: <physical_interface_name> <5>
+    type: ethernet
+    state: up
+# ...
+----
+<1> Defines the DNS server for your bare-metal system.
+<2> Replace `<dns_ip_address>` with the IP address for the DNS server.
+<3> Defines the bridge interface and its static IP configuration.
+<4> Replace `<gateway_ip>` with the IP address of the gateway.
+<5> Details the physical interface that you set as the bridge port.
+
+. Apply the network configuration from the YAML file to the network interfaces for the host by entering the following command: 
 +
 [source,terminal]
 ----
-$ sudo nohup bash -c "
-    nmcli con down \"$PUB_CONN\"
-    nmcli con delete \"$PUB_CONN\"
-    # RHEL 8.1 appends the word \"System\" in front of the connection, delete in case it exists
-    nmcli con down \"System $PUB_CONN\"
-    nmcli con delete \"System $PUB_CONN\"
-    nmcli connection add ifname baremetal type bridge con-name baremetal bridge.stp no ipv4.method manual ipv4.addr "x.x.x.x/yy" ipv4.gateway "a.a.a.a" ipv4.dns "b.b.b.b" <1>
-    nmcli con add type bridge-slave ifname \"$PUB_CONN\" master baremetal
-    nmcli con up baremetal
-"
-----
-<1> Replace `<con_name>` with the connection name. Replace `x.x.x.x/yy` with the IP address and CIDR for the network. Replace `a.a.a.a` with the network gateway. Replace `b.b.b.b` with the IP address of the DNS server. 
+$ nmstatectl apply --no-commit <path_to_network_yaml> <1>
+----
+<1> The `--no-commit` option tests connectivity for 60 seconds so if a commit timeout happens under 60 seconds, the network configuration rollbacks to an earlier state.
 
-. Optional: If you are deploying with a provisioning network, export the provisioning network NIC name by running the following command:
+. Verify the NMState syntax of the network file by entering the following command. Ensure that you check the command output for any errors:
```

Comment:
This doesn't accomplish much. NMState will check the syntax before applying the config anyway. We need gc more for things like day one network config where NMState isn't immediately being run on it and errors might take a long time to be found.